packages/repo-tools/package.json

```diff
@@ -84,17 +86,22 @@
     "@backstage/types": "workspace:^",
     "@types/is-glob": "^4.0.2",
     "@types/node": "^20.16.0",
-    "@types/prettier": "^2.0.0"
+    "@types/prettier": "^2.0.0",
+    "embedded-postgres": "17.2.0-beta.15"
```

Comment:
Did the beta have something in particular you needed, or could we be on the mainline instead?

Reply:
There aren't any mainline releases, they're all marked as beta for now: https://www.npmjs.com/package/embedded-postgres?activeTab=versions